src/fs-utils.js

```diff
@@ -0,0 +1,41 @@
+import { copy } from 'fs-extra';
+import { Promise } from 'bluebird';
+import temp from 'temp';
+import fs from 'fs';
+
+const log = require('debug')('electron-windows-installer:fs-utils');
+
+temp.track();
+
+const promisedCopy = Promise.promisify(copy);
+const promisedCreateTempDir = Promise.promisify(temp.mkdir);
+const readFileAsync = Promise.promisify(fs.readFile);
+const readDirAsync = Promise.promisify(fs.readdir);
+const unlinkAsync = Promise.promisify(fs.unlink);
+const writeFileAsync = Promise.promisify(fs.writeFile);
+const renameAsync = Promise.promisify(fs.rename);
+const inspectAsync = Promise.promisify(fs.stat);
+
+export {
```

Comment:
Your exposing an api that all return promises - there is no need to name them all Async.. if you had to you could call the file fs-promise-utils or something but i think thats unnneccessary, its internal you may as well leave it out..